core/model/modx/processors/search/search.class.php

```diff
@@ -185,7 +188,9 @@ public function searchResources()
                 'description' => $record->get('description'),
                 'type' => $type,
                 'type_label' => $typeLabel,
+                'icon' => $record->get('icon')?:false,
             );
+            if($this->results['icon'])$this->results['icon']=str_replace('icon-','',$this->results['icon']);
```

Comment:
Please, do not use such short `if` statements. It is difficult to read such code and find errors in case of any issues in the future.